website/docs/r/ui_plugin.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_ui_plugin"
+sidebar_current: "docs-vcd-resource-ui-plugin"
+description: |-
+  Provides a VMware Cloud Director UI Plugin resource. This can be used to manage UI Plugins.
+---
+
+# vcd\_ui\_plugin
+
+Provides a VMware Cloud Director UI Plugin resource. This can be used to manage UI Plugins in VCD, for example to add a new
+plugin from an local ZIP file, to publish/unpublish a UI Plugin to different Organizations, etc.
+
+-> Creating, updating and deleting UI Plugins requires System Administrator privileges.
```

Comment:
This statement seems to signify that reading doesn't require System Administrator privileges. But then the data source says that sys admin privileges are also required for reading.
If that's correct, can we say "all operations with UI plugins require System Administrator privileges"?

Reply:
Yeah, you're right. To be honest I don't know why Reading is missing there or why I made that specific mention of operations. I have changed it.